Pipfile.lock

```diff
@@ -12,1093 +12,1067 @@
+                "sha256:1866cf2c284a03b9524a5cc00daca56d80057c5ce3cdc86a52020f4c720856f0",
+                "sha256:3fb9575a5acd13031c57a62cc7823e5d2ff8bc3835ba4d94b921b4e6ee664104",
+                "sha256:825d5fccef6da42f3c8eccd4281af399f21c02b32d98e113dbc631ea6a6ecbc7",
+                "sha256:4ff604ce439abb20794f05613c374759ce10e3595d1867764dd1ae675b85acbd"
             ],
-            "markers": "python_version < '3.11' and platform_python_implementation == 'CPython'",
+            "index": "pypi-org-simple",
+            "markers": "platform_python_implementation == \"CPython\" and python_version < \"3.11\"",
             "version": "==0.2.6"
         },
         "sentry-sdk": {
             "hashes": [
-                "sha256:1ab34e3851a34aeb3d1af1a0f77cec73978c4e9698e5210d050e4932953cb241",
-                "sha256:ac2a50128409d57655279817aedcb7800cace1f76b266f3dd62055d5afd6e098"
+                "sha256:c17179183cac614e900cbd048dab03f49a48e2820182ec686c25e7ce46f8548f",
+                "sha256:6c01d9d0b65935fd275adc120194737d1df317dce811e642cbf0394d0d37a007"
             ],
-            "version": "==1.5.6"
+            "index": "pypi-org-simple",
+            "version": "==1.5.11"
+        },
+        "setuptools": {
+            "hashes": [
+                "sha256:69cc739bc2662098a68a9bc575cd974a57969e70c1d58ade89d104ab73d79770",
+                "sha256:f10059f0152e0b7fb6b2edd77bcb1ecd4c9ed7048a826eb2d79f72fd2e6e237b",
+                "sha256:69cc739bc2662098a68a9bc575cd974a57969e70c1d58ade89d104ab73d79770",
+                "sha256:f10059f0152e0b7fb6b2edd77bcb1ecd4c9ed7048a826eb2d79f72fd2e6e237b"
+            ],
+            "index": "pypi-org-simple",
+            "version": "==58.0.4"
         },
         "six": {
             "hashes": [
-                "sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926",
-                "sha256:8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254"
+                "sha256:8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254",
+                "sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
             ],
-            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
+            "index": "pypi-org-simple",
             "version": "==1.16.0"
         },
         "thoth-analyzer": {
             "hashes": [
-                "sha256:3f830334a3ba725cacf64ccc756e42f0c7946fd8038da6565cb2de569ea5c9c1",
-                "sha256:8a29ce615e5feddd301a8c6656132268bede23d4187a5cfb60f790f80cd04dc1"
+                "sha256:66e3561ae1ba01ef7ad15a255b866c5eccce6f445a5bef21096dcbc756d65be6",
+                "sha256:097d537bc21904c41d16c22bd389f909c2c5f397570696558de4baefa79d09f8"
             ],
-            "index": "pypi",
-            "version": "==0.1.8"
+            "index": "pypi-org-simple",
+            "version": "==0.0.7"
```

Comment:
Hmmm... version [0.0.7](https://github.com/thoth-station/analyzer/tree/0.0.7) is over 4 years old.

Should we _not_ have this downgrade here?